src/Microsoft.ML.Fairlearn/metrics/FairlearnMetricCatalog.cs

```diff
@@ -0,0 +1,291 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Data.Analysis;
+using Microsoft.ML.Data;
+using Microsoft.ML.Fairlearn.metrics;
+
+namespace Microsoft.ML.Fairlearn
+{
+    public class FairlearnMetricCatalog
```

Comment:
Can you add comments for all the public things?

Reply:
And in the documentation can you talk about any limits this has? Like currently needing all data to fit in memory.